docs/src/examples/multiple_shooting.md

```diff
@@ -66,7 +67,12 @@ callback = function (p, l, preds; doplot = true)
 
 	# plot the different predictions for individual shoot
 	plot_multiple_shoot(plt, preds, group_size)
-
+        
+	# plot the single prediction using multiple shoot
+	#prediction  = solve(remake(prob_node, p=p, tspan=(tsteps[1],tsteps[datasize]),u0=ode_data[:,1]), solver, saveat=tsteps)
+	#plot!(plt, tsteps, prediction[1,:],label = "Prediction")
```

Comment:
I don't see much point in adding commented out code, why not actually run and show the plots with some description?